src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -464,14 +466,20 @@ export const ConsultationDetails = (props: any) => {
               ></ABGPlots>
             </div>
           )}
-          {tab === "NURSING" && (
+          {tab === "IO%20BALANCE" && (
```

Comment:
```suggestion
          {tab === "IO_BALANCE" && (
```

Reply:
`IO_BALANCE` is not working because there is a space between words.